index.html

```diff
@@ -2250,22 +2273,22 @@ <h3>Refreshing</h3>
 The <a>issuer</a> can include the refresh service as an element inside the
 <a>verifiable credential</a> if it is intended for either the <a>verifier</a>
 or the <a>holder</a> (or both), or inside the <a>verifiable presentation</a>
-if it is intended for the <a>holder</a> only. In the latter case, this enables 
-the <a>holder</a> to refresh the <a>verifiable credential</a> before creating 
-a <a>verifiable presentation</a> to share with a <a>verifier</a>. In the former case, 
-including the refresh service inside the <a>verifiable credential</a> enables both 
+if it is intended for the <a>holder</a> only. In the latter case, this enables
+the <a>holder</a> to refresh the <a>verifiable credential</a> before creating
+a <a>verifiable presentation</a> to share with a <a>verifier</a>. In the former case,
+including the refresh service inside the <a>verifiable credential</a> enables both
 the <a>holder</a> and the <a>verifier</a> to perform future updates of the credential.
         </p>
         <p>
-The refresh service is only expected to be used when either the credential 
-has expired, or when the issuer does not publish credential status information. 
-It is ill advised for issuers to put the <code>refreshService</code> in a 
-<a>verifiable credential</a> that does not contain public information, 
-or whose refresh service is not protected in some way. 
+The refresh service is only expected to be used when either the credential
+has expired, or when the issuer does not publish credential status information.
```

Comment:
```suggestion
has expired or when the issuer does not publish credential status information.
```

Reply:
for clarity and grammar...

"when either the credential has expired or when the issuer" 

-> "when either (1) the credential has expired or (2) the issuer"
or
-> "either when the credential has expired or when the issuer" 

I prefer the former.

Reply:
Fixed in https://github.com/w3c/vc-data-model/pull/547/commits/34c6c5c061cc1a418e3579fce46003d38f392655.